tests/spec/core/utils-spec.js

```diff
@@ -8,6 +8,56 @@ describe("Core - Utils", () => {
     });
   });
 
+  async function clearCaches(){
```

Comment:
should this function not go in describe block below?

Reply:
Good point. Will move it.